README.md

```diff
@@ -14,3 +14,48 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+## Answer
+
+## Recurrence Relation and Summation Analysis
+
+Let T(n) be the time complexity of the algorithm for an input of size n.
+
+The algorithm divides the problem into subproblems of size $n/2^i$, where $i$ goes from $1$ to $log_2(n)$. At each level, we perform $n * 2^i$ comparisons.
+
+We can express this as a recurrence relation:
+
+$$ T(n) = \sum_{i=1}^{\log_2(n)} (n * 2^i) $$
+
+$$ T(n) = n * \sum_{i=1}^{\log_2(n)} (2^i) $$
+
+This looks like we can use the geometric series here.
+
+$$
+\begin{aligned}
+\sum_{i=1}^{\log_2(n)} [2^i] &= 2 * \frac{2^{\log_2(n)} - 1}{2 - 1} \\
+&= 2 * (n - 1) \\
+&= 2n - 2
+\end{aligned}
+$$
+
+Now I can substitute back in the original equation.
+
+
+
+$T(n) = n * (2n - 2)$
+
+$= 2n^2 - 2n$
+
+
+
+This simplifies down to $T(n) = \Theta(n^2)$
```

Comment:
A T(n) expression cannot be equal to an asymptotic bound. Result also incorrect.